components/body/index.jade

```diff
@@ -57,11 +57,11 @@
       .right-gutter-formatter
         p.body-section-sub-header= copy.partnership.gallery.subTitle[language]
         p.format-spacing= copy.partnership.gallery.text[language]
-        button.avant-garde-button-black= copy.partnership.gallery.link[language]
+        button.avant-garde-button-black(onclick="location.href='https://www.artsy.net/galleries';")= copy.partnership.gallery.link[language]
```

Comment:
I don't think any of this stuff should be nested under a `chevron-nav-list`: that's just for a list of flat links that look a specific way. Of which this doesn't seem to be?